src/main/asciidoc/_chapters/ops_mgt.adoc

```diff
@@ -2519,12 +2533,12 @@ The next time the cleaning process needs to look for a log, it starts by using i
 NOTE: WALs are saved when replication is enabled or disabled as long as peers exist.
 
 [[rs.failover.details]]
-==== Region Server Failover
+==== Region Server Failover(based on ZooKeeper)
```

Comment:
Here we do not need to se two different sections. First, we could mention the 'setting a watcher' way, this is way in the old time.

And starting from 2.5.0, the failover logic has been moved to SCP, where we add a `SERVER_CRASH_CLAIM_REPLICATION_QUEUES` step in SCP to claim the replication queues for a dead server.

And starting from 3.0.0, where we changed the replication queue storage from zookeeper to table, the update to the replication queue storage is async, so we also need an extra step to add the missing replication queues before claiming.

And on how to claim the replication queue, you can have two sections, to describe the layout and claiming way for zookeeper based implementation and table based implementation.